extensions/interactions/ImageClickInput/directives/ImageClickInput.js

```diff
@@ -47,8 +51,31 @@ oppia.directive('oppiaInteractiveImageClickInput', [
               '/' + encodeURIComponent($scope.filepath)) : null);
           $scope.mouseX = 0;
           $scope.mouseY = 0;
+          $scope.interactionIsActive = true;
           $scope.currentlyHoveredRegions = [];
+          $scope.previousInput = $scope.getLastAnswer();
           $scope.allRegions = imageAndRegions.labeledRegions;
+
+          $scope.checkPointerInRegion = function() {
```

Comment:
$scope.updateCurrentlyHoveredRegions

(The current function name suggests that this should return something.)

Reply:
Done